remo/featuredrep/templates/featuredrep_alter.jinja

```diff
@@ -33,6 +33,11 @@
         </span>
         {{ field_errors(form.users) }}
       </div>
+      <div class="large-6 columns">
+        <input type="text" class="flat" value="{{ form.created_on.value()|default("", True) }}"
+            name="created_on" placeholder="YYYY-MM-DD HH:MM:SS" />
```

Comment:
Instead of using `input` here you can use the field_with_attrs(form.created_on).

Reply:
I didn't know that, I will do asap

Reply:
For reference, the `field_with_attrs` is defined here:
https://github.com/mozilla/remo/blob/master/remo/base/templatetags/helpers.py#L228

Reply:
I made that change